api/errorHandling.go

```diff
@@ -0,0 +1,10 @@
+package api
+
+import "github.com/gin-gonic/gin"
+
+func checkErrorOrAbort(ctx *gin.Context, code int, err error) (success bool) {
```

Comment:
Could we name name it something like isSuccessfulOrAbort or hasNoErrorOrAbort, currently it is not so clear IMO.